.circleci/config.yml

```diff
@@ -43,6 +49,11 @@ jobs:
       - run: # start early for less wait time below
           command: ./scripts/cosm/start.sh
           background: true
+      - attach_workspace:
+          at: /tmp/builds
+      - run:
+          name: Merge build folders into project (merge with hardlinks)
+          command: cp --recursive --link /tmp/builds/* .
```

Comment:
interesting command, something i wished to do before but didn't know how

Reply:
I wonder if it is really possible to hardlink from the "mounted" workspace into the guest's main system. Depending on the implementation, those might be two different file systems. Anyways, it works and does the copy step in less than 1 second.